moveit_experimental/jog_arm/config/sia5_simulated_config.yaml

```diff
@@ -0,0 +1,33 @@
+use_gazebo: true # Whether the robot is started in a Gazebo simulation environment
+check_collisions: true # Check collisions?
+command_in_type: "unitless" # "unitless"> in the range [-1:1], as if from joystick. "speed_units"> cmds are in m/s and rad/s
+scale: # Only used if command_in_type=="unitless"
+  linear:  0.003  # Max linear velocity. Meters per publish_period.
+  rotational:  0.006  # Max angular velocity. Rads per publish_period.
+  joint: 0.01  # Max joint angular/linear velocity. Rads or Meters per publish period.
+cartesian_command_in_topic:  jog_arm_server/delta_jog_cmds # Topic for xyz commands
+joint_command_in_topic: jog_arm_server/joint_delta_jog_cmds # Topic for angle commands
+command_frame:  base_link  # If incoming cmds have no frame, this is the default
+incoming_command_timeout:  5  # Stop jogging if X seconds elapse without a new cmd
+joint_topic:  joint_states
```

Comment:
Any reason why you prefer that? To me, that would add another file that needs to be edited by the user... I'd prefer to keep all of the editable parameters in one place.

Reply:
Remapping can be done from the command line as well (using `input:=output` syntax).

Otherwise `.launch` files are used with `remap` elements.

Any serious use of ROS nodes/applications already means creating `.launch` files, so relying on `remap`ping to configure input and output topics would not seem like additional overhead.

Furthermore, remapping is considered best-practice for this, while configuring topic/service/action names in a `.yaml`/`.ini` file is an anti-pattern (`.launch` files can be parsed by external tools and `remap` is a standard element that any parser should understand. A custom `.yaml` file with custom key-value pairs is going to break that).

Reply:
I'm going to ignore this suggestion for now. In my mind, ease of use outweighs these other concerns.